spec/models/lifetime_usage_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe LifetimeUsage, type: :model do
+  subject(:lifetime_usage) { create(:lifetime_usage) }
+
+  it { is_expected.to belong_to(:organization) }
+  it { is_expected.to monetize(:current_usage_amount_cents) }
+  it { is_expected.to monetize(:invoiced_usage_amount_cents) }
+
+  describe 'default scope' do
+    let(:deleted_lifetime_usage) { create(:lifetime_usage, :deleted) }
+
+    it "only returns non-deleted lifetime-usage objects" do
+      expect(LifetimeUsage.all).to eq([lifetime_usage])
```

Comment:
be aware the `deleted_lifetime_usage` record hasn't been created, `let` is lazy and only executes the code when is called. Same for the next line.

you can add the following code to ensure the records are created to properly setup the test

```
before do
  lifetime_usage
  deleted_lifetime_usage
end
```

Reply:
Yeah don’t understand, I remember writing `let!` here. But will make sure it’s created in advance.

Reply:
maybe rubocop does not like `let!`

Reply:
Appears it was just an oversight, fixed it.